tools/functional-tester/etcd-runner/command/election_command.go

```diff
@@ -49,6 +50,8 @@ func runElectionFunc(cmd *cobra.Command, args []string) {
 	eps := endpointsFromFlag(cmd)
 	dialTimeout := dialTimeoutFromCmd(cmd)
 
+	elector := stringutil.RandomStrings(10, 1)[0]
```

Comment:
s/elector/election since it's a name of a particular election. Electors are participants.